rocketpy/AeroSurface.py

```diff
@@ -576,14 +576,14 @@ def evaluate_lift_coefficient(self):
             )
 
             # Differentiating at x = 0 to get cl_alpha
-            clalpha2D_mach0 = self.airfoil_cl.differentiate(x=1e-3, dx=1e-3)
+            clalpha2D_alpha0 = self.airfoil_cl.differentiate(x=1e-3, dx=1e-3)
```

Comment:
This change is wrong here. The `clalpha2D_mach0` is a float, and is independent of the mach number. We then apply a correction to make it a function of `mach` and call it `clalpha2D`

Also, since we are using Barrowman the `clalpha` is constant at all angle of attack

Reply:
The previous naming is not that good, but its there just to differentiate `clalpha2D_mach0` form `clalpha2D`